extensions/sources/sql/sql.json

```diff
@@ -60,6 +60,36 @@
           "zh_CN": "间隔时间"
         }
       },
+      {
+        "name": "maxRetry",
```

Comment:
These two props have the potential to be common properties. Try to make sure the names are the same across all sources.